index.html

```diff
@@ -1,16 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vanilla Todo</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-
-<body>
-    <div class="container"></div>
-</body>
-<script src="script.js"></script>
+  </head>
 
-</html>
\ No newline at end of file
+  <body>
+    <div class="container">
+      <!-- Title -->
+      <h1 style="border-bottom: solid; padding-bottom: 1rem">
+        오늘 해야할 일이 뭐가 있을까 ?
+      </h1>
+      <!-- Pending List Title -->
+      <div
+        class="pending"
+        style="
+          font-size: 1.3rem;
+          font-weight: bold;
+          text-align: left;
+          margin-left: 1rem;
+        "
+      >
+        해야할 것들
+      </div>
+      <!-- Pending List -->
+      <div class="pending-list">
+        <ul class="toDos" type="square"></ul>
+      </div>
+      <!-- Done List Title -->
+      <div
+        class="done"
+        style="
+          font-size: 1.3rem;
+          font-weight: bold;
+          text-align: left;
+          margin-left: 1rem;
+        "
+      >
+        다했다~
+      </div>
+      <!-- Done List -->
+      <div class="done-list">
+        <ul class="dones" type="square"></ul>
+      </div>
+      <form class="inputForm">
```

Comment:
저도 이번에 리뷰를 받아 알게 된 사실인데,
html의 일반적인 네이밍 컨벤션은 kebab-case를 사용한다고 하더라구요😀
```suggestion
      <form class="input-form">
```